dataprep/eda/basic/compute.py

```diff
@@ -331,12 +332,15 @@ def compute_univariate(
             else:
                 print("Invalid range of values for this column", file=stderr)
         data_num: List[Any] = []
+
+        df_x = drop_null(df[x])
+
         # qq plot
-        qqdata = calc_qqnorm(df[x].dropna())
+        qqdata = calc_qqnorm(df_x)
         # kde plot
-        kdedata = calc_hist_kde(df[x].dropna().values, bins)
+        kdedata = calc_hist_kde(df_x.values, bins)
         # box plot
-        boxdata = calc_box(df[[x]].dropna(), bins, dtype=dtype)
+        boxdata = calc_box(df_x, bins, dtype=dtype)
         # histogram
```

Comment:
`df_x` is a series but `df[[x]].dropna()` is a dataframe, and `calc_box` only takes a dataframe as input. Can you please do `df_x.to_frame()` and do some testing to make sure it works?

Reply:
made the changes!